src/taco-cli/test/templateManager.ts

```diff
@@ -361,4 +304,85 @@ describe("TemplateManager", function (): void {
                 });
         });
     });
+
+    describe("copyTemplateItemsToProject()", function (): void {
+        var testProjectPath: string = path.join(runFolder, "testProject");
+
+        beforeEach(function (): void {
+            rimraf.sync(testProjectPath);
+            wrench.mkdirSyncRecursive(testProjectPath, 511); // 511 decimal is 0777 octal
+        });
+
+        it("should copy items to the project when the template has a www folder", function (done: MochaDone): void {
+            var templates: templateManager = new templateManager(mockKitHelper);
+            var cordovaParameters: Cordova.ICordovaCreateParameters = {
+                appId: "test",
+                appName: "test",
+                cordovaConfig: {},
+                projectPath: testProjectPath,
+                copyFrom: testTemplate2Src
```

Comment:
How about testTemplatePath & testTemplateWithWwwPath instead of testTemplateSrc and testTemplate2Src respectively, for readability?